examples/all-clusters-app/esp32/main/main.cpp

```diff
@@ -135,18 +135,22 @@ void AddDevice(std::string name)
 
 class EditAttributeListModel : public ListScreen::Model
 {
-    int d;
-    int e;
-    int c;
-    int a;
+    int endPointIndex;
```

Comment:
Just note that d e c a stood for device endpoint cluster attribute
and the new names do not correspond.
Ensure that is what is intended.

Reply:
doh! I will fix this. Thank you!